Description: The node.js binary is called nodejs in Debian.
Author: Felix Geyer <fgeyer@debian.org>

--- a/ext/common/Constants.h
+++ b/ext/common/Constants.h
@@ -58,7 +58,7 @@
 
 	#define DEFAULT_MAX_POOL_SIZE 6
 
-	#define DEFAULT_NODEJS "node"
+	#define DEFAULT_NODEJS "nodejs"
 
 	#define DEFAULT_POOL_IDLE_TIME 300
 
--- a/lib/phusion_passenger/constants.rb
+++ b/lib/phusion_passenger/constants.rb
@@ -37,7 +37,7 @@ module PhusionPassenger
 		DEFAULT_LOG_LEVEL = 0
 		DEFAULT_RUBY = "ruby"
 		DEFAULT_PYTHON = "python"
-		DEFAULT_NODEJS = "node"
+		DEFAULT_NODEJS = "nodejs"
 		DEFAULT_MAX_POOL_SIZE = 6
 		DEFAULT_POOL_IDLE_TIME = 300
 		DEFAULT_START_TIMEOUT = 90_000
